### DISCUSSION TITLE
How to delay deps validation error display?

### DISCUSSION THREAD

**Discussion by JunyeongChoi0** [react-hook-form/react-hook-form#9274] [Q&A] [answered]
We can use `delayError` option for delaying error display.

https://codesandbox.io/s/rhf-interactive-field-forked-2u2nof?file=/src/App.js

See above sandbox with 1000ms delayError.

If you change 'desired' value as 2, error message appears instantly under 'max' input and under 'desired' 1s after.

Is this intended action?

If so, how to let deps validation error message appear at same time with original error message?

**Comment by leapful** [comment, marked as answer]
@JunyeongChoi0 

You can apply debounce on method "onChange" like this discussion https://github.com/react-hook-form/react-hook-form/discussions/7676#discussioncomment-2057674

Working demo: https://codesandbox.io/s/rhf-interactive-field-forked-3gmq9c?file=/src/App.js

**Reply by jordan-rigo** [reply]
There is an issue with this one, if the user submits the form before the onChange event runs, that creates an unpleasent situation.
Is there anyway to go around it?
